client/test_methods.go

```diff
@@ -229,6 +302,120 @@ func RequestCrumb(roundTripper *mhttp.MockRoundTripper, rootURL string) (
 	return
 }
 
+// RequestUpdateCenter only for the test case
+func RequestUpdateCenter(roundTripper *mhttp.MockRoundTripper, rootURL string) (
```

Comment:
Consider all test methods start with `PrepareFor`.

Reply:
I see the method `RequestCrumb` no start with `PrepareFor` and some methods start with `Prepare`，not `PrepareFor`